test/helpers/module.py

```diff
@@ -22,34 +22,34 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
-
+from importlib import import_module, reload
 import sys
 
-from importlib import reload
-
 
-def reload_plugin(name):
+def reload_plugin(name, replace_in=None):
     """
-    Reload builtin plugin module, e.g. `NotifyGnome`.
-    set filename to plugin to be reloaded (for example NotifyGnome.py)
+    Reload built-in plugin module, e.g. `NotifyGnome`.
 
-    The following libraries need to be reloaded to prevent
-     TypeError: super(type, obj): obj must be an instance or subtype of type
-     This is better explained in this StackOverflow post:
-        https://stackoverflow.com/questions/31363311/\
-          any-way-to-manually-fix-operation-of-\
-             super-after-ipython-reload-avoiding-ty
+    Reloading plugin modules is needed when testing module-level code of
+    notification plugins.
 
+    See also https://stackoverflow.com/questions/31363311.
     """
 
     module_name = f"apprise.plugins.{name}"
 
     reload(sys.modules['apprise.common'])
     reload(sys.modules['apprise.attachment'])
     reload(sys.modules['apprise.config'])
-    if module_name in sys.modules:
-        reload(sys.modules[module_name])
+    reload(sys.modules[module_name])
```

Comment:
That might be dangerous because it raises a `KeyError` when the module has not been loaded yet.

Reply:
I think it's better they way it was honestly.  There is no test coverage in a test file, so it's not like it matters at the end of the day.

Reply:
I've just reverted this change.